ambry-router/src/main/java/com/github/ambry/router/GetBlobOperation.java

```diff
@@ -219,6 +316,12 @@ private void onChunkOperationComplete(GetChunk chunk) {
     if (chunk == firstChunk) {
       if (operationCallbackInvoked.compareAndSet(false, true)) {
         if (chunk.chunkBlobId.getBlobDataType() == BlobId.BlobDataType.METADATA) {
+          /**
+           * Save metadata on successful completion. Everything is decrypted at this point.
+           */
```

Comment:
I realize that you have a lot of comments that are using /** */. we should just use // for multiple lines.

Reply:
done